packs/mmonit/sensors/events_sensor.yaml

```diff
@@ -0,0 +1,22 @@
+---
+  class_name: "MmonitEventsSensor"
+  entry_point: "events_sensor.py"
+  description: "Sensor which monitors mmonit for events"
+  poll_interval: 10
+  trigger_types:
+    -
+      name: "new_event"
+      description: "Trigger which indicates that a new event has arrived at mmonit"
+      payload_schema:
+        type: "object"
+        properties:
+          host:
+            type: "string"
+          id:
+            type: "integer"
+          event:
```

Comment:
Oh I get it now. The problem is the events could be any type actually, I was thinking about fields instead. Unfortunately the event type depends on the type of check monit does (process, url, mysql, ssh, file check, checksum,....) and there are way too many types to set them in there, at least in a consistent manner.